app/assemblers/assemble_representation_tracks_for_select.rb

```diff
@@ -0,0 +1,24 @@
+class AssembleRepresentationTracksForSelect
+  include Mandate
+
+  initialize_with :mentor, :status
```

Comment:
As there are only two statuses, I'd prefer a `with_feedback: ` here I think. Saves for typos with symbols. Unless you think there might ever be a third status?